packages/react-core/src/components/Alert/examples/Alert.md

```diff
@@ -414,7 +414,7 @@ Users navigating via keyboard or another assistive technology will need a way to
 
 The following example shows how alerts can be triggered by an asynchronous event in the application. You can customize how an alert will be announced to assistive technology by adjusting the value of the `aria-live` property. Click the "start async" alert button below and then click the buttons in the above toast examples to demonstrate how asynchronous events add alerts to a group. Click the "stop async alerts" button to halt this behavior.
 
-See the [alert group accessibility tab](/components/alert-group/accessibility) for more information on customizing this behavior.
+See the [alert accessibility tab](/components/alert/accessibility) for more information on customizing this behavior.
```

Comment:
This link is still broken

Reply:
This link is correct - but the accessibility doc is not available in the react workspace. it's only available when react workspace is consumed in org and built along with the accessibility docs.